src/test/java/PageBasis.java

```diff
@@ -0,0 +1,11 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class PageBasis {
+    public WebDriver driver;
```

Comment:
Why are fields public?

Reply:
Same as before - just missed this moment.